packages/core/src/tests/date-input/date-input.e2e.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Siemens AG
```

Comment:
```suggestion
 * SPDX-FileCopyrightText: 2025 Siemens AG
```

File was created in 2025.